app/(main)/newchat/components/step1.tsx

```
@@ -58,6 +58,7 @@ import axios from "axios";
 import getHeaders from "@/app/utils/headers.util";
 import ParsingProgress from "./ParsingProgress";
 import FileSelector, { ParseFilters } from "./FileSelector";
+import FileSelector, { ParseFilters } from "./FileSelector";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'FileSelector'. Consider to delete it or rename it.

'FileSelector' is defined here:


(lint/suspicious/noRedeclare)